src/pages/electricity.tsx

```diff
@@ -1,17 +1,42 @@
 import Link from 'next/link';
 import Head from 'next/head';
+import { trpc } from '../utils/trpc';
+import ElectricityForm from '../components/Electricity/ElectricityForm';
+import { UnregisteredElectricityRequest } from '../schema/electricity.schema';
+import ElectricityResponse from '../components/Electricity/ElectricityResponse';
+import { Spinner } from 'flowbite-react';
+import wateringPlantsImage from '../assets/images/watering-plants-img.png';
+import Image from 'next/image';
 
 export default function ElectricityConsumption() {
+  const { mutate, data, isLoading } = trpc.useMutation([
+    'electricity.unregistered-request-electricity',
+  ]);
+  const handleSubmit = (electricityData: UnregisteredElectricityRequest) => {
+    mutate({ ...electricityData });
+  };
+
+  if (isLoading) {
+    return (
+      <div className="mx-auto">
```

Comment:
When I ran this locally, the spinner was not centered. Maybe change the classes to 'flex items-center mt-5' just show it centers on the page with a bit of margin on the top?

Reply:
Yeah it did look weird, decided to move it after where the picture is rendered and added `justify-center` so hopefully looks a little better!